app/customer.py

```diff
@@ -0,0 +1,76 @@
+import math
+from dataclasses import dataclass
+
+
+from app.shop import Shop, print_check
+
+
+@dataclass
+class Customer:
+    name: str
+    product_cart: dict
+    location: list
+    money: int
+    car: dict
+
+
+def cost_of_km(customer: Customer, data: dict) -> int | float:
+    return (customer.car["fuel_consumption"] / 100) * data["FUEL_PRICE"]
+
+
+def cost_by_distance(

```

Comment:
you should rewrite these functions into methods and then call them on the instance of a class